tools/coverity/misra.config

```diff
@@ -38,6 +42,10 @@
             deviation: "Rule 8.9",
             reason: "For ease, configuration parameters are defined at a global scope even when used only once."
         },
+        {
+            deviation: "Rule 8.13",
```

Comment:
did you have a look at how this rule has been successfully suppressed inline in the +tcp library - https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/dcd0df708af3083d9a72bf3154caf038dacf6dbd/source/FreeRTOS_IP.c#L235

Reply:
I messaged him about this offline. Marking as resolved.